<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunTimeInformation</class>
 <widget class="QMainWindow" name="RunTimeInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>39</y>
      <width>321</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Predators</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="predatorGroupsBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Groups Information</string>
       </property>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>30</y>
          <width>281</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="bestPredatorBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Best Creature</string>
       </property>
       <widget class="QLabel" name="labelLifePredator">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Life</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStrenghPredator">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Strengh</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelVisionPredator">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Vision</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelSpeedPredator">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelPolicyPredator">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Policy</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelGroupIDPredator">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>GroupID</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatusPredator">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>70</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>39</y>
      <width>321</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Preys</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="preyGroupsBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Groups Information</string>
       </property>
       <widget class="QListView" name="listViewPrey">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>30</y>
          <width>281</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="bestPreyBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Best Creature</string>
       </property>
       <widget class="QLabel" name="labelStrenghPrey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Strengh</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelLifePrey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Life</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelVisionPrey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Vision</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelPolicyPrey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Policy</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelSpeedPrey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatusPrey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>70</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelIteration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteration</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Per Iteration</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
